### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by alonamid**
**Related issue**: 

**Type of change**: other enhancement

**Impact**: other

**Release Notes**

Update CHANGELOG for 1.2.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+* Ring Topology System Bus NoC (#461 )
+* Add option to use blackboxed SimDRAM instead of SimAXIMem (#449)
+* Log `init-submodules` script (#433 )
+* Integration of the Ariane core into Chipyard (#448)
+* Moved the Docker image used for CI into Chipyard (prev. in BOOM) (#463)
+
+### Changed
+* IOBinders and BuildTop unification between FireChip and Chipyard (#390)
+* Update to CircleCI 2.1 config. syntax and cleanup CI file (#421)
+* Bump FireSim to version 1.9 - Includes FirePerf TracerV stack unwinding features
```
Comment (jerryz123) [inline CHANGELOG.md]:
```suggestion
* Bump FireSim to version 1.8.1 - Includes FirePerf TracerV stack unwinding features
```

Comment (alonamid) [reply CHANGELOG.md]: AFAIK, the decision was to keep it 1.9 @sagark @davidbiancolin

Comment (sagark) [reply CHANGELOG.md]: yes, but this should be 1.9.0

Comment (sagark) [inline CHANGELOG.md]:
```suggestion
* Bump FireSim to version 1.9.0 - Includes FirePerf TracerV stack unwinding features
```

**Pull request merged**